doc/modules/model_evaluation.rst

```diff
@@ -85,6 +85,7 @@ Scoring                           Function
 **Regression**
 'explained_variance'              :func:`metrics.explained_variance_score`
 'neg_mean_absolute_error'         :func:`metrics.mean_absolute_error`
+'neg_mean_absolute_percentage_error' :func:`metrics.mean_absolute_percentage_error`
```

Comment:
@amueller suggested `neg_mape` here. I'm not entirely sure which is better. But you'll need to widen the table if we go with this name!

Reply:
Yes, I am open to change the name or keep it as is. But I'll the necessary changes once we agree on a name ;)

Reply:
@jnothman @amueller I am going to switch from the scorer fname `neg_mean_absolute_percentage_error` to the `neg_mape` since MAPE is already a famous acronym in the community and also because it's the only way to avoid pep8 problems.